### PR TITLE
Update readme/classifiers

### PR DESCRIPTION
**Pull request by jburel**
Clarify change in the url if installed with version prior to 5.2.6
This will require a version bump since the clarification should be pushed to pypi

I also have added classfiers

### CODE REVIEW

**Comment by will-moore** [comment]
Good to merge. However, I see that all the screenshots on https://github.com/ome/omero-gallery are failing to load from https://github.com/ome/omero-gallery/tree/gh-pages/images

**Comment by jburel** [comment]
The formatting used to display correctly on pypi is not well handled on gh.

**Comment by jburel** [comment]
@will-moore: back on.
gh had some issues earlier on

File: README.rst [hunk 1]
```diff
@@ -30,8 +34,9 @@ Add gallery custom app to your installed web apps:
 Now restart OMERO.web as normal.
 
 
-And you're done! Go to https://your-web-server/gallery
+**Warning**:
 
+OMERO.weberror version 2.x or newer is meant to be installed into OMERO.web **5.2.6 or newer**. This is due to Django Framework compatibility and a package reorganization required to distribute the application using Python Package Manager PyPi.
```
Comment (sbesson) [inline README.rst]: weberror -> gallery

Comment (jburel) [inline README.rst]: thanks

**Comment by jburel** [comment]
@sbesson: readme has been updated
I will relist for tomorrow for final check 
Also pushed on testpipy (ignored version) 
https://testpypi.python.org/pypi?:action=display&name=omero-gallery&version=2.0.10

To test
pip install -i https://testpypi.python.org/pypi omero-gallery

**Comment by aleksandra-tarkowska** [comment]
looks good:

```
(omerovenv) MacBookPro:OMERO.py ola$ pip install -i https://testpypi.python.org/pypi omero-gallery
Collecting omero-gallery
  Downloading https://testpypi.python.org/packages/4d/41/2e5e0fe0f0638c19f4e9373356ce108fcda97382a4b87e1e7877c4fe6263/omero-gallery-2.0.10.tar.gz (226kB)
    100% |████████████████████████████████| 235kB 4.5MB/s 
Building wheels for collected packages: omero-gallery
  Running setup.py bdist_wheel for omero-gallery ... done
  Stored in directory: /Users/ola/Library/Caches/pip/wheels/cd/63/88/85c7f1eb9660405f105ac9d02c728e50db9bae86fecd198bed
Successfully built omero-gallery
Installing collected packages: omero-gallery
Successfully installed omero-gallery-2.0.10
```

```
[11/Nov/2016 10:15:30] "GET /gallery/ HTTP/1.1" 200 3690
[11/Nov/2016 10:15:30] "GET /static/gallery/3rdparty/bootstrap/css/bootstrap.min.css HTTP/1.1" 200 99961
[11/Nov/2016 10:15:30] "GET /static/gallery/3rdparty/bootstrap/js/bootstrap.min.js HTTP/1.1" 200 29110
[11/Nov/2016 10:15:31] "GET /webgateway/render_thumbnail/3/96/ HTTP/1.1" 200 3181
[11/Nov/2016 10:15:38] "GET /gallery/show_group/9/ HTTP/1.1" 200 70119
```

**Comment by aleksandra-tarkowska** [comment]
@jburel could you release please

**Comment by jburel** [comment]
pypi:
https://pypi.python.org/pypi?:action=display&name=omero-gallery&version=2.0.1

tag:
https://github.com/ome/omero-gallery/tree/v2.0.1

**Pull request merged**